### PR TITLE
Dd 13 refactor jira issue

### PR DESCRIPTION
**Pull request by ghost**
Break Jira Issue file into multiple files and make a hierarchy to avoid duplicating code

### CODE REVIEW

File: DailyReportWeb/Controllers/Api/ReportController.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+﻿using Equilobe.DailyReport.Models.Interfaces;
+using System.Web.Http;
+
+namespace DailyReportWeb.Controllers.Api
+{
+    [Authorize]
+    public class ReportController : ApiController
+    {
+        public IReportService ReportService { get; set; }
+
```
Comment (Sebyd) [inline DailyReportWeb/Controllers/Api/ReportController.cs]: Let's also add the method types:
HttpGet("{id}") and HttpPost("{id}")

File: Equilobe.DailyReport.BL/Jira/SprintLoader.cs [hunk 2]
```diff
@@ -24,9 +24,9 @@ public SprintLoader(ProjectDateFilter filter, JiraClient client)
 
         public SprintContext GetSprintDetails()
         {
-            var boardId = Client.Board(Filter.ProjectKey).Id;
+            var boardId = Client.Board(Filter.ProjectKey).id;
```
Comment (Sebyd) [inline Equilobe.DailyReport.BL/Jira/SprintLoader.cs]: Should be "Id" not "id"

File: Equilobe.DailyReport.Models/Jira/Status.cs [hunk 3]
```diff
@@ -0,0 +1,28 @@
+﻿using System.Runtime.Serialization;
+
+namespace Equilobe.DailyReport.Models.Jira
+{
+    [DataContract]
+    public class Status : JiraIdentifiableResponse
+    {
+        public static Status UNKNOWN_STATUS = new Status()
```
Comment (Sebyd) [inline Equilobe.DailyReport.Models/Jira/Status.cs]: Is this actually used? If not, we could remove it.

**Review by Sebyd — CHANGES_REQUESTED** [review]

File: DailyReportWeb/Controllers/Api/ReportController.cs [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+        [HttpGet]
```
Comment (Sebyd) [inline DailyReportWeb/Controllers/Api/ReportController.cs]: you need to put the id in the route. HttpGet("{id}")

Comment (ghost) [reply DailyReportWeb/Controllers/Api/ReportController.cs]: it doesn't work. I get an error and from what I've seen on stackoverflow it's related to the fact that it's needed web api 2.0

**Review by Sebyd — APPROVED** [review]

**Pull request merged**